style.css

```diff
@@ -1,14 +1,25 @@
+/* COLOUR HEX CODES:
```

Comment:
Great choice in colours, keeping with the theme of static TV having no signal 💯

Reply:
This could have the :root { } selector for the colours to be contained in ⭐  great work 😄

Reply:
This could have the :root { } selector for the colours to be contained in ⭐  great work 😄